presto-docs/src/main/sphinx/presto_cpp/properties-session.rst

```diff
@@ -258,3 +258,13 @@ The maximum size in bytes for the task's buffered output. The buffer is shared a
 The maximum bytes to buffer per PartitionedOutput operator to avoid creating tiny SerializedPages.
 For PartitionedOutputNode::Kind::kPartitioned, PartitionedOutput operator would buffer up to that number of
 bytes / number of destinations for each destination before producing a SerializedPage. Default is 32MB.
+
+``native_max_local_exchange_partition_count``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``bigint``
+* **Default value:** ``4294967296``
+
+Maximum number of partitions created by a local exchange.
+Affects concurrency for pipelines containing LocalPartitionNode.
+Defaults to 4294967296.
```

Comment:
```suggestion
```
Suggest deleting this sentence because it repeats the information in line 266 and does not express the number in a different way that is more easily human-readable. For example, line 260 restates the default value for ``native_max_page_partitioning_buffer_size`` from ``33554432`` to ``32MB``.

Reply:
Good catch. Fixed.